### PR TITLE
pAI expansion

### PR DESCRIPTION
**Pull request by VasilisThePikachu**
Design doc to have more feature to the pAI

### CODE REVIEW

File: src/en/proposals/pai-expansion.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+[VasilisThePikachu, Unapproved]
+
+This design doc will outline some new ideas for the pAI item and ghost role. As of now they are just chatbots that can play music and quickly can become boring for both the pAI and to their owner. Especially if one or both of them does not rp.
+
+This will give pAI's more options of stuff they can do. Thus making them a more convincing option for players.
+
+These are ideas taken from TG
+
+### (Optional) Hologram
+pAI's can choose to manifest themselves as a hologram in the world. They will be unable to intaract with the world in any way and can only go a certain small tile radius away. Could be nice to intaract with their owner or attract attention on to their dead owner. They will slightly
```
Comment (deltanedas) [inline src/en/proposals/pai-expansion.md:12]: They will slightly :trollface:

Comment (VasilisThePikachu) [reply src/en/proposals/pai-expansion.md]: Took me a good minute to notice what you wanted.

**Comment by revsys413** [comment]
i’ve been thinking about this for a while and was considering working on the programs reusing revenant code until i saw this pr—one idea someone i was talking to brought up was a chem scanner, which would work something like chem goggles but built into the pai

**Comment by VasilisThePikachu** [comment]
> i’ve been thinking about this for a while and was considering working on the programs reusing revenant code until i saw this pr—one idea someone i was talking to brought up was a chem scanner, which would work something like chem goggles but built into the pai

That could work too, i can add it

**Comment by squirrelanna** [comment]
Something to consider is that a pAI made from a potato would probably not have most of the basic features, so a way to synthesize those features and add them to the pAI is probably necessary. Maybe there could be pAI specific modules that science can make similar to borg modules?

**Comment by squirrelanna** [comment]
A few different ideas:

1. Flashlight. While not particularly bright, it would be placeable and allow for just a little more visibility when needed. Letting the pAI toggle it on and off would also allow for quickly toggling to indicate something urgent visually. Optional: RGB lighting.

2. Recorder. Adds the ability for the pAI to transcribe conversations into text files, limited by time so that it can't be left on indefinitely. Useful for quick, automated notetaking when receiving orders. Particularly useful for security to transcribe interviews/interrogations and potentially as a bodycam of sorts. Potential upgrade: The ability to also record emotes in addition to speech. Maybe even give visual playback that can be played on a TV similar to how reporter cameras work.

**Comment by revsys413** [comment]
here’s a thought—roundstart pAI work normally and have the program system and whatnot, but science and salv can research/find program modules for potatoes

for sci i’m thinking it’d probably be included with the relevant borg research

**Comment by mirrorcult** [comment]
please include in summary.md & standardize in line with the [feature proposal reference](https://docs.spacestation14.com/en/proposals/proposal-template.html)

**Comment by VasilisThePikachu** [comment]
Done!

**Comment by LankLTE** [comment]
@VasilisThePikachu Do you still want this document reviewed?

**Comment by VasilisThePikachu** [comment]
yes please

**Comment by VasilisThePikachu** [comment]
Look at #183 instead

**Comment by deltanedas** [comment]
wtf real

**Pull request closed without merging**